packages/site/pages/index.mdx

```diff
@@ -30,7 +30,7 @@ import Link from 'next/link'
 <section style={{['--brand-Card-maxWidth']: '100%',maxWidth: '800px',margin: '0 auto', ['--brand-Grid-spacing-row']: 'var(--brand-Grid-spacing-column-gap)'}}>
   <Grid>
     <Grid.Column span={12}>
-      <Link legacyBehavior href="/getting-started/introduction">
+      <Link legacyBehavior passHref href="/getting-started/introduction">
```

Comment:
Does `passRef` need to be added if legacyBehavior is there? I thought it was one or the other?